deploy.sh

```diff
@@ -1 +1,79 @@
-#!/bin/bash
\ No newline at end of file
+#!/bin/bash
+
+# Added color constants
+RED='\033[0;31m'
+NC='\033[0m'
+
+check_if_exists_func()
+{
+  get_resource=$(kubectl get "$1" | grep "$2")
```

Comment:
Could we use name of the resource directly to prevent any unpredictable behavior? Or is the grep needed?

```suggestion
  get_resource=$(kubectl get "$1" "$2" --no-headers 2>/dev/null)
```